src/server.ts

```diff
@@ -380,16 +375,18 @@ const createParticipant = async (tag: string): Promise<ParticipantInfo> => {
 const deleteSession = async () => {
   if (sessionId) {
     try {
+      await webRTCController.deleteSession(accountId, sessionId);
       console.log(`Deleted WebRTC session: ${sessionId} `);
       sessionId = "";
     } catch (e) {
-      let error: any = e;
+      let error: ApiResponse<void> = e as ApiResponse<void>;
```

Comment:
Did this end up working (tested?), or is this supposed to be `ApiError`

Reply:
might be worth doing something like 

```
if (e instanceof ApiError) {
        const error: ApiError = e
        // do logging
}
```

Reply:
I tested it and it worked, although it exposed the fact that the error report for the delete participant in the original code would never have worked (and has been fixed)